spec/system/claims/schools/users/invite_a_user_to_a_school_spec.rb

```diff
@@ -7,180 +7,341 @@
     perform_enqueued_jobs { example.run }
   end
 
-  let(:school) { create(:claims_school, :claims, urn: "123456") }
-  let(:anne) { create(:claims_user, :anne) }
-  let(:another_school) { create(:claims_school, :claims) }
-
-  before do
-    setup_school_and_anne_membership
-  end
-
-  scenario "I sign in as a lead mentor user and invite a user to a school" do
-    sign_in_as_lead_mentor_user
-    visit_claims_school_users_page
-    when_i_click("Add user")
-    fill_in_user_details
-    check_user_details
-    when_i_click("Confirm and add user")
-    verify_user_added
-  end
-
-  scenario "I sign in as a lead mentor user and enter invalid user details" do
-    sign_in_as_lead_mentor_user
-    visit_claims_school_users_page
-    when_i_click("Add user")
-    fill_in_invalid_user_details
-    then_see_error_message
-  end
-
-  scenario "I try to add a user who already exists" do
-    sign_in_as_lead_mentor_user
-    visit_claims_school_users_page
-    when_i_click("Add user")
-    fill_in_user_details
-    check_user_details
-    when_i_click("Confirm and add user")
-    and_user_is_added
-    when_i_click("Add user")
-    fill_in_user_details
-    then_see_error_message_for_existing_user
-  end
-
-  scenario "I CANT access another schools users list" do
-    sign_in_as_lead_mentor_user
-    verify_i_cant_access_another_schools_users_list
-  end
-
-  scenario "I use back or change to edit my answers" do
-    sign_in_as_lead_mentor_user
-    visit_claims_school_users_page
-    when_i_click("Add user")
-    fill_in_user_details
-    check_user_details
-    click_back
-    check_form_is_populated
-    edit_first_name
-    check_user_details_updated
-    when_i_click("Confirm and add user")
-    verify_edited_user_added
+  scenario "Claims user invites and views a user" do
+    given_a_school_exists_with_claims_users
+    and_i_am_signed_in
+
+    when_i_navigate_to_the_claim_school_users_page
+    then_i_see_the_claim_users_page
+    and_i_select_add_user
+    then_i_see_the_add_user_page
+
+    when_i_fill_in_invalid_user_email_details
+    and_i_select_continue
```

Comment:
```suggestion
    and_i_click_on_continue
```
The language is slightly wrong here. We reserve `select` for selecting options on a form, such as dropdowns, radios, and checkboxes. 
Click more accurately describes clicking on a button.